test/jdk/sun/java2d/Disposer/TestDisposerRace.java

```diff
@@ -40,30 +41,35 @@ public final class TestDisposerRace {
     private static volatile boolean disposerDone = false;
 
     public static void main(String[] args) throws Exception {
-        TestDisposerRace test = new TestDisposerRace();
-        test.run();
+        new TestDisposerRace().run();
 
         checkRecordsCountIsSane();
         if (recordsCount.get() > 0) {
+            System.err.println("Some records (" + recordsCount + ") have not been " + "disposed");
```

Comment:
If allocating the exception on the next line fails due to OOME, hopefully this message got printed to the log first.